app/assets/javascripts/clipboard.js

```diff
@@ -0,0 +1,6 @@
+$(document).ready(function(){
+	var clip = new ZeroClipboard($("#clip_button"))
```

Comment:
Missing semicolon.<br>'ZeroClipboard' is not defined.<br>'clip' is defined but never used.